src/MTGCore/ViewModels/SearchFilter.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System.Collections.Generic;
+
+namespace MTGCore.ViewModels
+{
+    public class SearchFilter
+    {
+        public string Name { get; set; }
+        public string Type { get; set; }
+        public string Rarity { get; set; }
```

Comment:
Again, it might be worth storing rarities as enums? That way, we restrict the values to only the ones we want and it might be handy in the future?